### DISCUSSION TITLE
Greenfield: entry boxes with more obvious depth

### DISCUSSION THREAD

**Discussion by hanaral** [elementary/stylesheet#667] [Ideas]
I'm certain that you already aware of it, but entry boxes are lacking depth, and I dont just mean the colouring. I think that there should be some form of shading (like the old stlyesheet) where it looks like it curves inwards
For example, in the original stylesheet I could tell it was a text entry box right away because of the subtle depth:
![image](https://user-images.githubusercontent.com/58219504/81478304-4ef67f80-9214-11ea-8bb3-902c7e864c86.png)
I'm afraid that I will have to glance twice to identify the type of box it is with out said depth, so I hope you are able to implement it in some way

**Comment by danirabbit** [comment]
I’m hesitant to continue using a gradient here because it doesn’t really scale:

Text view widgets look awkward with a gradient since they’re multi-line. So it seems kind of odd that two widgets that both accept text would be styled differently.

We already have workarounds for when entries are linked vertically to remove the gradient since it looks odd to have two undulations butted up against each other.

So it kind of feels like this wasn’t actually a good design choice

**Comment by hanaral** [comment]
Alright, I didn't take in to consideration multi line boxes.
I'm not sure if this is already on the way, but would it be possible to have the inset shadow to increase slightly when interacting with the box? I think that would achieve a similar goal, and would probably scale much better

**Comment by hanaral** [comment]
Apparently GH could use some better UX; I can't delete my own discussion.
@cassidyjames Apparently maintainers can delete these thingies, could someone do the honours please 😓️
